modules/standard/CTypes.chpl

```diff
@@ -788,6 +788,72 @@ module CTypes {
   extern proc c_pointer_diff(a:c_ptr(void), b:c_ptr(void),
                              eltSize:c_ptrdiff):c_ptrdiff;
 
+  /*
+    Toggles whether the new or deprecated behavior of :proc:`c_ptrTo` and
+    :proc:`c_ptrToConst` is used for :type:`~String.string`,
+    :type:`~Bytes.bytes`, and class type arguments. (The behavior of
+    :proc:`c_ptrTo` with array type arguments is unaffected.)
+
+    The new behavior is to return a :type:`c_ptr`/:type:`c_ptrConst` to the
+    underlying buffer of the ``string`` or ``bytes``, or to the heap instance of
+    a class type. The deprecated behavior is to return a
+    :type:`c_ptr`/:type:`c_ptrConst` to the ``string`` or ``bytes`` itself, or
+    the stack representation of the class — this matches the behavior of
+    :proc:`c_addrOf`/:proc:`c_addrOfConst`.
+
+    The deprecated behavior is on by default. To opt in to the new behavior,
```

Comment:
it looks like the other places in this doc use `opt-in` rather than `opt in`, consider updating this use for consistency